src/app/page.tsx

```diff
@@ -37,32 +42,39 @@ const SelectSystemContainer = styled.div`
   margin: 16px 0;
 `;
 
+const ProductListContainer = styled.div`
+  margin: 16px 0;
+`;
+
 const SelectSystem: React.FC = () => {
   const systems = useMetaIntegratedSystemsList();
   const router = useRouter();
-
+  const [_selectedSystem, setSelectedSystem] = useState<string | null>(null);
   const hndSystemChange = (ev: React.ChangeEvent<HTMLSelectElement>) => {
+    setSelectedSystem(ev.target.value);
     router.push(`/?system=${ev.target.value}`);
   };
 
   return (
-    <SelectSystemContainer>
-      {systems.isFetched && systems.data ? (
-        <>
-          <label htmlFor="system">Select a system:</label>
-          <select name="system" id="system" onChange={hndSystemChange}>
-            <option value="">Select a system</option>
-            {systems.data.results.map((system) => (
-              <option key={system.id} value={system.slug || ""}>
-                {system.name}
-              </option>
-            ))}
-          </select>
-        </>
-      ) : (
-        <p>Loading systems...</p>
-      )}
-    </SelectSystemContainer>
+    <>
```

Comment:
This doesn't need to be wrapped, it can just be the SelectSystemContainer component.